.gitignore

```diff
@@ -362,3 +362,6 @@ tilt-settings.json
 
 # dummy provider's certificate directory
 test/externaldata/dummy-provider/certs/
+
+# annotations for dapr
+annotations.yaml
```

Comment:
i think we added this to .tmp, do we still need this?

Reply:
I reverted that change back, since creating `annotation.yaml` in `.tmp` was causing CI to error out with `cannot create /.tmp/annotations.yaml: Directory nonexistent` msg.

Reply:
Why is the directory nonexistent? Is there a way to remedy this?

It would be good to avoid clutter in the root directory: saying "`rm -rf <temp directory>` will recover if we have any bad state" is a lot easier than saying "delete these X files"

Reply:
bump (was not addressed since last review request)

Reply:
My bad. I addressed it. Creating a tmp dir while bootstrapping to mitigate `dir nonexistent` error.

Reply:
Awesome, thanks!